packages/sdks-tests/src/tests/blocks.spec.ts

```diff
@@ -92,6 +99,72 @@ test.describe('Blocks', () => {
     }
   });
 
+  test('video', async ({ page }) => {
+    test.skip(isRNSDK);
+    const mockVideoPath = path.join(__dirname, '..', 'mocks', 'video.mp4');
+    const mockVideoBuffer = fs.readFileSync(mockVideoPath);
+
+    await page.route('**/*', route => {
+      const request = route.request();
+      if (request.url().includes(VIDEO_CDN_URL)) {
+        route.fulfill({
+          status: 200,
+          contentType: 'video/mp4',
+          body: mockVideoBuffer,
+        });
+      } else {
+        route.continue();
+      }
+    });
+
+    await page.goto('/video');
+
+    const videoLocator = page.locator('video');
+
+    const expectedCSS: Record<string, string>[] = [
+      {
+        width: '152px',
+        'object-fit': 'cover',
+        'z-index': '2',
+        'border-radius': '1px',
+        position: 'absolute',
+      },
+      {
+        width: '1249px',
+        'object-fit': 'contain',
+        'z-index': '2',
+        'border-radius': '1px',
+        position: 'absolute',
+      },
+      {
+        width: '744px',
+        'object-fit': 'cover',
+        'z-index': '2',
+        'border-radius': '1px',
+        position: 'absolute',
+      },
+      {
+        width: '152px',
+        'object-fit': 'cover',
+        'z-index': '2',
+        'border-radius': '1px',
+        position: 'absolute',
+      },
+    ];
+
+    await expect(videoLocator).toHaveCount(expectedCSS.length);
+
+    const expectedVals = expectedCSS.map((val, i) => ({ val, i }));
+
+    for (const { val, i } of Object.values(expectedVals)) {
+      const video = videoLocator.nth(i);
+      const expected = val;
+      for (const property of Object.keys(expected)) {
+        await expect(video).toHaveCSS(property, expected[property]);
+      }
+    }
+  });
```

Comment:
That's awesome! Very well done 😄.

One last thing would be to add a test that proves your PR works as intended! From what I can tell, you are providing a child `<p>asfgasgasgasg some test</p`. Could you check that it is visible in the page? 🙏🏽

Reply:
Thank you!! I've added the test that shows that the children are visible and rendered, please let me know if any changes are needed to be made